mytransactions.sql

```diff
@@ -0,0 +1,30 @@
+BEGIN;
+UPDATE animals
+SET species = 'unspecified';
+ROLLBACK;
+
+BEGIN;
+UPDATE animals 
+SET species = 'digimon'
+WHERE name LIKE '%mon'; 
+UPDATE animals 
+SET species = 'pokemon'
+WHERE species is NULL; 
+COMMIT;
+
+BEGIN;
+DELETE FROM animals;
+ROLLBACK;
```

Comment:
- [ ] As per this [requirement](https://github.com/microverseinc/curriculum-databases/blob/main/db-structure/vet_clinic_query_update_tables.md#:~:text=After%20the%20rollback%20verify%20if%20all%20records%20in%20the%20animals%20table%20still%20exists), Please verify if all records in the `animals` table still exist after the rollback

Reply:
Hi, @Omar-Muhamad  this was also verified by the screenshot of the third transaction in the pull request.